src/lib.rs

```diff
@@ -722,6 +722,22 @@ impl<'poll> Tiny<'poll> {
                 );
             }
 
+            Cmd::CAP { client: _, ref subcommand, ref params } => {
+                match subcommand.as_ref() {
+                    "NAK" => {
+                        let msg_target = MsgTarget::Server {
+                            serv_name: conn.get_serv_name(),
+                        };
+                        if params.iter().any(|cap| cap.as_str() == "sasl") {
+                            self.tui.add_err_msg("Server does not support SASL authenication",
+                                                 Timestamp::now(),
+                                                 &msg_target);
+                        }
+                    }
+                    &_ => {}
```

Comment:
Why not just `_`?

Could you also log this case? We should try to log unsupported responses from servers as much as possible, it makes debugging and extending much easier. (check for log lines in `handle_msg()` as example)

Reply:
changed and logged (to debug log).